#! /bin/sh /usr/share/dpatch/dpatch-run
## 25-remove-duplicate-files.dpatch by  <pgquiles@elpauer.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove duplicate .pidl and .h files, which fail to install the second 
## DP: time if using recent version of '/usr/bin/install'. This is somewhat 
## DP: fixed upstream (they are fixing the generated Makefile.am)
## DP: and will be part of 5.7.6.

@DPATCH@

diff -rupd ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/RTEvent.mpc ACE_wrappers/TAO/orbsvcs/orbsvcs/RTEvent.mpc
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/RTEvent.mpc	2009-12-11 19:21:41.629361106 +0100
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/RTEvent.mpc	2009-12-11 19:33:50.797407833 +0100
@@ -31,7 +31,6 @@ project(RTEvent) : orbsvcslib, orbsvcs_o
     RtecEventChannelAdminC.h
     RtecUDPAdminC.h
     Event/event_export.h
-    Event/event_skel_export.h
   }
 
   Inline_Files {
diff -rupd ACE_wrappers.orig/TAO/tao/BiDir_GIOP.mpc ACE_wrappers/TAO/tao/BiDir_GIOP.mpc
--- ACE_wrappers.orig/TAO/tao/BiDir_GIOP.mpc	2009-12-11 19:21:41.885362571 +0100
+++ ACE_wrappers/TAO/tao/BiDir_GIOP.mpc	2009-12-11 19:25:14.804361835 +0100
@@ -23,10 +23,6 @@ project : taolib, tao_output, install, p
     BiDir_GIOP
   }
 
-  PIDL_Files {
-    BiDir_GIOP
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd -Sci -SS -Sorb -Sa -St \
                 -Wb,export_macro=TAO_BiDirGIOP_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/CodecFactory.mpc ACE_wrappers/TAO/tao/CodecFactory.mpc
--- ACE_wrappers.orig/TAO/tao/CodecFactory.mpc	2009-12-11 19:21:41.853361325 +0100
+++ ACE_wrappers/TAO/tao/CodecFactory.mpc	2009-12-11 19:24:59.312681157 +0100
@@ -23,10 +23,6 @@ project : taolib, tao_output, install, c
     CodecFactory
   }
 
-  PIDL_Files {
-    CodecFactory
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd -Sci -SS -Sorb -Sal -GX \
                 -Wb,export_macro=TAO_CodecFactory_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/Compression.mpc ACE_wrappers/TAO/tao/Compression.mpc
--- ACE_wrappers.orig/TAO/tao/Compression.mpc	2009-12-11 19:21:41.845361444 +0100
+++ ACE_wrappers/TAO/tao/Compression.mpc	2009-12-11 19:28:47.445372018 +0100
@@ -23,10 +23,6 @@ project : taolib, tao_output, install, t
     Compression
   }
 
-  PIDL_Files {
-    Compression
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd -Sci -SS -Sorb -Sa -St \
                 -Wb,export_macro=TAO_Compression_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/CSD_Framework.mpc ACE_wrappers/TAO/tao/CSD_Framework.mpc
--- ACE_wrappers.orig/TAO/tao/CSD_Framework.mpc	2009-12-11 19:21:41.841361203 +0100
+++ ACE_wrappers/TAO/tao/CSD_Framework.mpc	2009-12-11 19:28:34.133364466 +0100
@@ -28,10 +28,6 @@ project : taolib, tao_output, portablese
     CSD_Framework
   }
 
-  PIDL_Files {
-    CSD_Framework
-  }
-
   IDL_Files {
     idlflags += -Sci -SS -Gp -Gd -Sorb -GA -Sal \
                 -Wb,export_macro=TAO_CSD_FW_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/DynamicAny.mpc ACE_wrappers/TAO/tao/DynamicAny.mpc
--- ACE_wrappers.orig/TAO/tao/DynamicAny.mpc	2009-12-11 19:21:41.841361203 +0100
+++ ACE_wrappers/TAO/tao/DynamicAny.mpc	2009-12-11 19:26:21.296371313 +0100
@@ -36,10 +36,6 @@ project : install, tao_output, valuetype
     DynamicAny
   }
 
-  PIDL_Files {
-    DynamicAny
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd -Sci -SS -Sorb \
                 -Wb,export_macro=TAO_DynamicAny_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/IFR_Client.mpc ACE_wrappers/TAO/tao/IFR_Client.mpc
--- ACE_wrappers.orig/TAO/tao/IFR_Client.mpc	2009-12-11 19:21:41.897360892 +0100
+++ ACE_wrappers/TAO/tao/IFR_Client.mpc	2009-12-11 19:29:21.373362969 +0100
@@ -23,10 +23,6 @@ project : taolib, tao_output, install, c
     IFR_Client
   }
 
-  PIDL_Files {
-    IFR_Client
-  }
-
   IDL_Files {
     idlflags += -SS -GX \
                 -Wb,stub_export_macro=TAO_IFR_Client_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/ImR_Client.mpc ACE_wrappers/TAO/tao/ImR_Client.mpc
--- ACE_wrappers.orig/TAO/tao/ImR_Client.mpc	2009-12-11 19:21:41.889361051 +0100
+++ ACE_wrappers/TAO/tao/ImR_Client.mpc	2009-12-11 19:26:06.589361067 +0100
@@ -23,10 +23,6 @@ project : taolib, tao_output, install, p
     ImR_Client
   }
 
-  PIDL_Files {
-    ImR_Client
-  }
-
   IDL_Files {
     idlflags += -Gp -Ssi -Gd -Sorb -GA \
                 -Wb,export_macro=TAO_IMR_Client_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/IORManipulation.mpc ACE_wrappers/TAO/tao/IORManipulation.mpc
--- ACE_wrappers.orig/TAO/tao/IORManipulation.mpc	2009-12-11 19:21:41.845361444 +0100
+++ ACE_wrappers/TAO/tao/IORManipulation.mpc	2009-12-11 19:26:45.425391295 +0100
@@ -24,10 +24,6 @@ project : taolib, tao_output, install, c
     IORManipulation
   }
 
-  PIDL_Files {
-    IORManipulation
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd -Sci -SS -Sorb -GX \
                 -Wb,export_macro=TAO_IORManip_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/IORTable.mpc ACE_wrappers/TAO/tao/IORTable.mpc
--- ACE_wrappers.orig/TAO/tao/IORTable.mpc	2009-12-11 19:21:41.849361005 +0100
+++ ACE_wrappers/TAO/tao/IORTable.mpc	2009-12-11 19:29:46.001453664 +0100
@@ -24,10 +24,6 @@ project : taolib, tao_output, install, t
     IORTable
   }
 
-  PIDL_Files {
-    IORTable
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd -Sci -SS -Sorb \
                 -Wb,export_macro=TAO_IORTable_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/ObjRefTemplate.mpc ACE_wrappers/TAO/tao/ObjRefTemplate.mpc
--- ACE_wrappers.orig/TAO/tao/ObjRefTemplate.mpc	2009-12-11 19:21:41.885362571 +0100
+++ ACE_wrappers/TAO/tao/ObjRefTemplate.mpc	2009-12-11 19:29:08.069402948 +0100
@@ -23,10 +23,6 @@ project : valuetype, portableserver, ins
     ObjRefTemplate
   }
 
-  PIDL_Files {
-    ObjRefTemplate
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd -SS -Sorb -GX \
                 -Wb,export_macro=TAO_ORT_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/PI.mpc ACE_wrappers/TAO/tao/PI.mpc
--- ACE_wrappers.orig/TAO/tao/PI.mpc	2009-12-11 19:21:57.576388948 +0100
+++ ACE_wrappers/TAO/tao/PI.mpc	2009-12-11 17:21:54.000000000 +0100
@@ -27,10 +27,6 @@ project : taolib, tao_output, codecfacto
     PI
   }
 
-  PIDL_Files {
-    PI
-  }
-
   IDL_Files {
     idlflags += -Sorb -GX \
                 -Wb,unique_include=tao/PI/PI.h
diff -rupd ACE_wrappers.orig/TAO/tao/PI_Server.mpc ACE_wrappers/TAO/tao/PI_Server.mpc
--- ACE_wrappers.orig/TAO/tao/PI_Server.mpc	2009-12-11 19:21:41.841361203 +0100
+++ ACE_wrappers/TAO/tao/PI_Server.mpc	2009-12-11 19:25:53.381362130 +0100
@@ -32,10 +32,6 @@ project : taolib, tao_output, pi, portab
     PI_Server
   }
 
-  PIDL_Files {
-    PI_Server
-  }
-
   IDL_Files {
     idlflags += -Sci -SS -GA -Gp -Gd -Sorb -Sal
 //                -Wb,include_guard=TAO_PI_SERVER_SAFE_INCLUDE \
diff -rupd ACE_wrappers.orig/TAO/tao/RTCORBA.mpc ACE_wrappers/TAO/tao/RTCORBA.mpc
--- ACE_wrappers.orig/TAO/tao/RTCORBA.mpc	2009-12-11 19:21:41.833361761 +0100
+++ ACE_wrappers/TAO/tao/RTCORBA.mpc	2009-12-11 19:22:30.309416956 +0100
@@ -28,10 +28,6 @@ project : taolib, tao_output, install, p
     RTCORBA
   }
 
-  PIDL_Files {
-    RTCORBA
-  }
-
   IDL_Files {
     idlflags += -Sa \
                 -Wb,unique_include=tao/RTCORBA/RTCORBA.h
diff -rupd ACE_wrappers.orig/TAO/tao/RTPortableServer.mpc ACE_wrappers/TAO/tao/RTPortableServer.mpc
--- ACE_wrappers.orig/TAO/tao/RTPortableServer.mpc	2009-12-11 19:21:41.845361444 +0100
+++ ACE_wrappers/TAO/tao/RTPortableServer.mpc	2009-12-11 19:25:40.052360908 +0100
@@ -28,10 +28,6 @@ project : taolib, tao_output, install, r
     RTPortableServer
   }
 
-  PIDL_Files {
-    RTPortableServer
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd \
                 -Wb,include_guard=TAO_RT_PORTABLESERVER_SAFE_INCLUDE \
diff -rupd ACE_wrappers.orig/TAO/tao/RTScheduler.mpc ACE_wrappers/TAO/tao/RTScheduler.mpc
--- ACE_wrappers.orig/TAO/tao/RTScheduler.mpc	2009-12-11 19:21:41.849361005 +0100
+++ ACE_wrappers/TAO/tao/RTScheduler.mpc	2009-12-11 19:24:48.088459307 +0100
@@ -23,10 +23,6 @@ project : taolib, tao_output, rtcorba, p
     RTScheduling
   }
 
-  PIDL_Files {
-    RTScheduling
-  }
-
   IDL_Files {
     idlflags += -Gp -Gd -Sci -SS -Sorb -GX \
                 -Wb,export_macro=TAO_RTScheduler_Export \
diff -rupd ACE_wrappers.orig/TAO/tao/tao.mpc ACE_wrappers/TAO/tao/tao.mpc
--- ACE_wrappers.orig/TAO/tao/tao.mpc	2009-12-11 19:21:57.576388948 +0100
+++ ACE_wrappers/TAO/tao/tao.mpc	2009-12-11 16:04:17.000000000 +0100
@@ -1,6 +1,6 @@
 //$Id: tao.mpc 79930 2007-11-05 03:06:19Z jtc $
 
-project(TAO) : acelib, install, tao_output, taodefaults, pidl, extra_core, taoidldefaults, gen_ostream, corba_e_micro, corba_e_compact, core_minimum_corba, tao_no_iiop {
+project(TAO) : acelib, install, tao_output, taodefaults, extra_core, taoidldefaults, gen_ostream, corba_e_micro, corba_e_compact, core_minimum_corba, tao_no_iiop {
   after       += TAO_Core_idl
   sharedname   = TAO
 
